src/selenium_screenshot.py

```diff
@@ -0,0 +1,134 @@
+import time
+
+from pyvirtualdisplay import Display
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+def _take_screenshot(browser, file_name):
+    img = browser.get_screenshot_as_png()
+    with open(file_name, "wb") as image:
+        image.write(img)
+
+
+def _init_browser(url, width=1000, height=500):
+    try:
+        window_width = int(width) if width is not None else 1000
+        window_height = int(height) if height is not None else 500
+        display = Display(visible=0, size=(1366, 768))
+        display.start()
+
+        chrome_options = webdriver.ChromeOptions()
+        chrome_options.add_argument("--no-sandbox")
+        chrome_options.add_argument("--disable-dev-shm-usage")
+
+        chrome_options.add_experimental_option("excludeSwitches", ["enable-automation"])
+        chrome_options.add_experimental_option(
+            "prefs", {"download.prompt_for_download": False}
+        )
+        browser = webdriver.Chrome(options=chrome_options)
+        browser.set_window_size(window_width, window_height)
+        browser.get(url)
+
+        return browser
+    except Exception as ex:
```

Comment:
Isn't this a violation of a pylint rule? The `broad-except` rule

Reply:
Yes, it is. But I don't know and I'm not sure which errors will be throw, then I preferred to use `Exception`.